tests/lib/collectors/osquery.py

```diff
@@ -87,6 +87,55 @@ def testProcess(self, mock_exists) -> None:
     self.assertEqual(len(containers), 1)
     self.assertEqual(containers[0].query, "SELECT * FROM processes")
 
+  @mock.patch('os.path.exists')
+  def testProcessQueryPack(self, mock_exists) -> None:
+    """Tests the collector's Process() function with a Osquery Pack file."""
+    test_state = state.DFTimewolfState(config.Config)
+    osquery_collector = osquery.OsqueryCollector(test_state)
+    mock_exists.return_value = True
+
+    test_ok_data = """{
+      "platform": "darwin",
+      "queries": {
+        "query_1": {
+          "query" : "select * from launchd where path like '%System.plist';",
```

Comment:
Thisis supposed to be a string, but the JSON withing should still be represented correctly. Another way to do this would be to just write a dict and call `json.dumps` on it to make it a string. 

```suggestion
          "query": "select * from launchd where path like '%System.plist';",
```